doc/source/user/quickstart.rst

```diff
@@ -868,9 +868,9 @@ copy.
     >>> def f(x):
     ...     print(id(x))
     ...
-    >>> id(a)  # id is a unique identifier of an object
+    >>> id(a)  # id is a unique identifier of an object #doctest: +SKIP
     148293216  # may vary
-    >>> f(a)
+    >>> f(a)   #doctest: +SKIP
```

Comment:
Why do we need `#doctest: +SKIP` if we already have `# may vary` for the output? This should work without the skip.

Reply:
This is linked to the fact that I was working toward a split refguide_check.py / pytest testing scenario. The `# may vary` is internal `refguide_check.py` cooking, pytest does not take it into consideration.